guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -3,41 +3,43 @@
 
 You can configure your {Project} to use two different modes of transport for remote job execution.
```

Comment:
I think its very important to make it crystal clear, a Smart Proxy can either use the push mode or the pull mode. Its not possible to have some hosts using pull and other push.

Reply:
> Its not possible to have some hosts using pull and other push.

For the sake of completeness, with a single proxy yes, but you can mix and match push and pull in a single job if you have multiple proxies configured to use different modes

Reply:
But that also depends on the setup of the host, doesn't it? What happens if the job selects a pull Proxy for a host that doesn't have the pull client?
I'm not sure if we should encourage users to mix them up.

Reply:
> What happens if the job selects a pull Proxy for a host that doesn't have the pull client?

Then the user has their environment set up incorrectly. If a user tells Foreman that host A is reachable through proxy P when it is not, then we can't really do anything about it.

>  I'm not sure we should encourage users to mix them up.

That is a different question. I was merely pointing out what is technically possible.